asset/src/elasticsearch_reader_api/reader.ts

```diff
@@ -0,0 +1,112 @@
+import elasticAPI from '@terascope/elasticsearch-api';
+import {
+    AnyObject,
+    DataEntity,
+    isObjectEntity,
+    getTypeOf,
+    Logger,
+    toNumber,
+} from '@terascope/job-components';
+import { CountParams, SearchParams } from 'elasticsearch';
+import { buildQuery } from '../elasticsearch_reader/elasticsearch_date_range/helpers';
+import { ESReaderOptions, SlicerDateResults } from '../elasticsearch_reader/interfaces';
+
+export default class APIReader {
+    readonly config: ESReaderOptions;
+    logger: Logger;
+    _baseClient: AnyObject;
+    client: elasticAPI.Client;
```

Comment:
We shouldn't be exposing the underlying client.

```suggestion
    protected readonly client: elasticAPI.Client;
```